### PR TITLE
feat: add upload backup ui page

### LINKED ISSUE #329: Create upload backup screen
UI implementation for uploading screen:

- [x] Add page for uploading backup with file dropzone
- [x] Support password check and errors

### PR DESCRIPTION
**Pull request by 0xmad**
## Explanation

This PR adds upload screen for backup.

Details are below:
- [x] Add upload input component
- [x] Add upload backup page
- [x] Add react-dropzone
- [x] Remove pretty-quick

## More Information

Closes #329

## Screenshots/Screencaps

N/A

## Manual Testing Steps

N/A

## Pre-Merge Checklist

- [x] PR template is filled out
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [x] PR is linked to the appropriate GitHub issue
- [x] PR has been added to the appropriate release Milestone

> PR template source from [github.com/MetaMask](https://github.com/MetaMask)

### CODE REVIEW

File: packages/app/src/ui/components/UploadInput/UploadInput.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          borderWidth: 2,
+          borderRadius: 2,
+          borderStyle: "dashed",
+          outline: "none",
+          cursor: "pointer",
+        }}
+      >
+        <input {...rest} {...getInputProps()} />
+
+        {isDragActive ? <p>Drop the files here...</p> : <p>Drop some files here, or click to select files</p>}
```
Comment (0xisk) [inline packages/app/src/ui/components/UploadInput/UploadInput.tsx:54]: Shouldn't it be just one file dropped? If so let's name it `file`.

File: packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx [hunk 2]
```diff
@@ -33,9 +33,9 @@ const DownloadBackup = (): JSX.Element => {
             <Typography variant="body1">Backup contains encrypted data:</Typography>
 
             <List dense>
-              <ListItem>- Your current password</ListItem>
+              <ListItem>- Your current password and mnemonic</ListItem>
 
-              <ListItem>- Your identities</ListItem>
+              <ListItem>- Your identities and accounts</ListItem>
```
Comment (0xisk) [inline packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx:38]: What is the difference between `identities` and `accounts`?

File: pnpm-lock.yaml
Comment (0xisk) [inline pnpm-lock.yaml:1]: It's not clear to me why all these changes in the pnpm-lock but I guess because this branch is not rebased yet with the main.

**Review by 0xisk — COMMENTED** [review]
Thanks @0xmad, just some minor comments.

File: packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx (hunk 2 above)
Comment (0xmad) [reply packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx:38]: With account user can sign the message and create an identity

Comment (0xisk) [reply packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx:38]: That might confuse users between `identities` and `accounts`. Could we maybe say `Ethereum accounts`?

Comment (0xisk) [reply packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx:38]: Yup that makes sense as well. :ok_hand:

**Review by 0xisk — APPROVED** [review]
Approved, thanks @0xmad!

**Review by 0xisk — APPROVED** [review]
Thanks @0xmad for fixing this.

**Pull request merged**